File: eCarService.Model/AdditionalService.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eCarService.Model
{
    public class AdditionalService
    {
        public int AdditionalServiceId { get; set; }
        public string Name { get; set; } 
        public string Description { get; set; }
        public string Status { get; set; }
        public decimal? Price { get; set; }
    }
}

File: ecarservice_mobile/lib/model/additionalService.dart
import 'package:json_annotation/json_annotation.dart';
part 'additionalService.g.dart';

@JsonSerializable()
class AdditionalService {
  int? additionalServiceId;
  String? name;
  String? description;
  double? price;

  AdditionalService() {}

  factory AdditionalService.fromJson(Map<String, dynamic> json) =>
      _$AdditionalServiceFromJson(json);

  /// Connect the generated [_$OfferToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$AdditionalServiceToJson(this);
}
